components/ui/DropDown/DropDown.tsx

```diff
@@ -0,0 +1,139 @@
+import { cva, VariantProps } from "class-variance-authority";
+import { Menu, Transition } from "@headlessui/react";
+import { Fragment } from "react";
+import { ChevronDownIcon } from "@heroicons/react/20/solid";
+
+const dropDownStyles = cva("", {
+  variants: {
+    items: {
+      primary: "bg-blue-200",
+      secondary: "bg-green-200",
+    },
+  },
+  defaultVariants: {
+    items: "primary",
+  },
+});
+export interface DropDownProps extends VariantProps<typeof dropDownStyles> {}
+
+export const DropDown = ({ items, ...props }: DropDownProps) => {
+  return (
+    <div className="fixed top-16 w-56 text-right">
```

Comment:
Let's take out the positioning of this element (top-16, w-56) and put it into the story.

Like [here](https://github.com/ninjha01/storybook/blob/b2275d3dc3accc68af584f70c5650cf0a2f35222/components/Blast/SequenceCard/SequenceCard.stories.tsx#L18)

Reply:
this is so that when we use the component in a new application, we won't have to fight the hardcoded positioning

Reply:
Without the positioning, when you open the dropdown the options get cutoff by the storyboard UI.